src/vue/modules/issuance-explorer/wrappers/issuance.js

```diff
@@ -1,4 +1,5 @@
 import _get from 'lodash/get'
+import { REQUEST_STATES } from '@/js/const/request-states.const'
 
 export class Issuance {
```

Comment:
Since we display only issuance requests, maybe should we rename this wrapper to `IssuanceRequest`?

Reply:
Make sense, in my opinion